templates/registration/edit_profile.html

```diff
@@ -0,0 +1,118 @@
+{% extends 'base.html' %}
+
+{% block title %}Edit your Profile{% endblock %}
+
+{% load static %}
+{% load is_login_signup_disabled %}
+{% block extra_scripts %}
+	<script src="{% static 'js/form-helpers.js' %}"></script>
+{% endblock %}
+
+
+{% block content %}
+<div class="container">
+    {% if not maintenance_mode|is_login_signup_disabled %}
+    <div class="card border-0 shadow-lg my-5">
+        <div class="card-body p-0">
+            <!-- Nested Row within Card Body -->
+            <div class="row">
+                <div class="col-xs-12 col-lg-8 offset-lg-2">
+                    <div class="px-3 py-5">
+                        <div class="row">
+                            <div class="col">
+								<div class="col">
+								<h2 class="text-dark mb-4 text-center">Edit your Profile</h2>
+                                </div>
+                            </div>
+                        </div>
+
+                        <div class="mw-30 m-auto pt-1">
+                            <form method="post">
+                                {% csrf_token %}
+                                <div class="row pb-3">
+                                    <div class="col-12">
+                                        <label class="form-label">{{form.email.label_tag}}</label>
+                                        {{form.email}}
+                                        <div class="form-text">{{form.email.help_text}}</div>
+                                        {% if form.email.errors %}
+                                        <div id="validation_email" class="invalid-feedback pt-1">
```

Comment:
Seems like there is no need for this if it's not editable

Reply:
And same goes for other fields

Reply:
Added errors except the 'unchanged' ones.